build.sbt

```diff
@@ -2,12 +2,13 @@ organization := "org.zalando"
 
 name := "play-etcd-watcher"
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.2"
```

Comment:
I would also suggest cross compilation. Just in case someone else would like to use it.